lib/ts/recipe/multifactorauth/types.ts

```diff
@@ -154,6 +154,7 @@ export type GetPhoneNumbersForFactorsFromOtherRecipesFunc = (
 
 export const FactorIds = {
     EMAILPASSWORD: "emailpassword",
+    WEBAUTHN: "webauthn",
```

Comment:
need to also modify the user object and login methods object (in ts/types.ts)

Reply:
Relate to that, will all webauthn methods have an email associated with that login method?

Reply:
> Relate to that, will all webauthn methods have an email associated with that login method?

don't think i understand.

> need to also modify the user object and login methods object (in ts/types.ts)
Isn't this enough? https://github.com/supertokens/supertokens-node/pull/951/files#diff-f8b1deda227dcaaaf5169c750f94e6e49ad2660d1064b9b46693d708a33e862fL198
As far as i can tell this is the only change needed .